src/drafts/SelectPanel2/SelectPanel.tsx

```diff
@@ -100,12 +99,41 @@ const SelectPanel = props => {
             setSearchQuery,
           }}
         >
-          <Box as="form" onSubmit={onInternalSubmit} sx={{display: 'flex', flexDirection: 'column', height: '100%'}}>
+          <Box
+            as="form"
+            onSubmit={onInternalSubmit}
+            sx={{
+              display: 'flex',
+              flexDirection: 'column',
+              height: '100%',
+            }}
+          >
             {/* render default header as fallback */}
             {slots.header || <SelectPanel.Header />}
-            {childrenInBody}
-            {/* render default footer as fallback */}
-            {slots.footer || <SelectPanel.Footer />}
+            <Box
+              sx={{
+                flexShrink: 1,
+                flexGrow: 1,
+                overflow: 'hidden',
+
+                display: 'flex',
+                flexDirection: 'column',
+                justifyContent: 'space-between',
+                ul: {overflowY: 'auto', flexGrow: 1},
+              }}
+            >
+              <ActionListContainerContext.Provider
+                value={{
+                  container: 'SelectPanel',
+                  listRole: 'listbox',
+                  selectionAttribute: 'aria-selected',
```

Comment:
From what I understand, the role is always going to be `listbox` and selectionAttribute be `aria-selected` for SelectPanel. is that right?

Reply:
Yep, that is always true :)